### PR TITLE
Fixes #17227 - Dashboard Content Host Subscription Status

### PR DESCRIPTION
**Pull request by ldjebran**
#17227 - Bugzilla:1180573 - url search parameter must be encoded (contain special chars)

### CODE REVIEW

**Comment by omaciel** [comment]
Thank you @ldjebran ! Could you [create an issue in Redmine](http://projects.theforeman.org/projects/katello/issues/new) too?

@thomasmckay can you indicate someone to review this for us?

**Comment by ldjebran** [comment]
@omaciel , issue created: http://projects.theforeman.org/issues/17227

**Comment by jlsherrill** [comment]
[test]

File: app/views/dashboard/_subscription_widget.html.erb [hunk 1]
```diff
@@ -1,3 +1,5 @@
+<% require 'erb' %>
```
Comment (jlsherrill) [inline app/views/dashboard/_subscription_widget.html.erb]: I don't think this is actually required?

Comment (jlsherrill) [inline app/views/dashboard/_subscription_widget.html.erb]: otherwise looks good :+1:

Comment (ldjebran) [inline app/views/dashboard/_subscription_widget.html.erb]: @jlsherrill, yes this work without it, just wanted to be sure that any other user should understand that we are are using ERB at first look, do you want me to remove it?

Comment (jlsherrill) [inline app/views/dashboard/_subscription_widget.html.erb]: Thats normally implied by the .erb extension.  I'm not aware of this being a normal convention, and it doesn't appear in any existing katello or foreman views, so I would recommend removing it.

**Comment by ldjebran** [comment]
@jlsherrill, change done

**Comment by jlsherrill** [comment]
[test]

**Comment by jlsherrill** [comment]
ACK thanks @ldjebran !  Merging!

**Pull request merged**